BraintreeCore/src/main/java/com/braintreepayments/api/InvalidArgumentException.kt

```diff
@@ -13,6 +13,7 @@ open class InvalidArgumentException : Exception {
      * @param message the error message
      * @suppress
      */
+    @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
     @Suppress("ConvertSecondaryConstructorToPrimary")
-    @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP) constructor(message: String?) : super(message)
+    constructor(message: String?) : super(message)
```

Comment:
why cant this constructor go next to the class name?

Reply:
Which part?

Reply:
This is an exception that we return to merchants sometimes so technically access to the class definition should be public, but we don't want merchants creating these exceptions so we restrict access to the constructor here.